### PR TITLE
[7.5] Log monitoring bulk failures (#14356)

### PR DESCRIPTION
**Pull request by ycombinator**
Backports the following commits to 7.5:
 - Log monitoring bulk failures  (#14356)

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -57,6 +57,13 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 - Fix checking tagsFilter using length in cloudwatch metricset. {pull}14525[14525]
 - Fixed bug with `elasticsearch/cluster_stats` metricset not recording license expiration date correctly. {issue}14541[14541] {pull}14591[14591]
+- Convert indexed ms-since-epoch timestamp fields in `elasticsearch/ml_job` metricset to ints from float64s. {issue}14220[14220] {pull}14222[14222]
+- Fix ARN parsing function to work for ELB ARNs. {pull}14316[14316]
+- Update azure configuration example. {issue}14224[14224]
+- Limit some of the error messages to the logs only {issue}14317[14317] {pull}14327[14327]
+- Fix cloudwatch metricset with names and dimensions in config. {issue}14376[14376] {pull}14391[14391]
+- Fix marshaling of ms-since-epoch values in `elasticsearch/cluster_stats` metricset. {pull}14378[14378]
+- Log bulk failures from bulk API requests to monitoring cluster. {issue}14303[14303] {pull}14356[14356]
```
Comment (ph) [inline CHANGELOG.next.asciidoc]: rebase issue?

Comment (ycombinator) [reply CHANGELOG.next.asciidoc]: Argh, yes, thanks!

Comment (ph) [reply CHANGELOG.next.asciidoc]: @ycombinator still an issue ? one day :robot: will take care of it.

**Comment by ycombinator** [comment]
Travis CI is green and Jenkins CI failures are unrelated. Merging.

**Pull request merged**